feathr_project/feathr/utils/dsl/functions.py

```diff
@@ -0,0 +1,183 @@
+SUPPORTED_FUNCTIONS = [
```

Comment:
There should be some comments to help people know this file is auto generated. Would be better to mention how it's generated.